README.md

```diff
@@ -87,3 +90,8 @@ Python-Specific Mappings
 \rf - run the current test
 \rl - run the last test
 ```
+
+Searching
+
+```
```

Comment:
I think you forgot to close these quotes.

Reply:
That I did!  Fixed.